CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to Elemental
+
+Elemental accepts contributions via GitHub issues and pull requests.
+This document outlines the process to get your pull request accepted.
+
+Elemental does accept external contributions in general, however the team's time is limited.
+
+ Elemental is on a quarterly release cycle, so it might take a long time
```

Comment:
Nit, heading space  (which would not even change the rendering I guess)
```suggestion
Elemental is on a quarterly release cycle, so it might take a long time
```